### PR TITLE
Add ImageCapture API entry

### PR DESCRIPTION
**Pull request by miguelao**

### CODE REVIEW

File: features-json/imagecapture.json [hunk 1]
```diff
@@ -0,0 +1,286 @@
+{
+  "title":"ImageCapture API",
+  "description":"The Image Capture API provides mechanisms to access the Video Camera and take photos, surfacing the camera capabilities and allowing",
```
Comment (Fyrd) [inline features-json/imagecapture.json]: "and allowing...?"

**Comment by Fyrd** [comment]
Looks good, thanks! Just seems like something's missing from the description. :)

**Comment by miguelao** [comment]
Oops, I took off half sentence :)
Finished wording, PTAL

**Comment by miguelao** [comment]
05195e5 correction: FF has IC behind a flag (re. the interchange with @NOtherDev in https://github.com/NOtherDev/whatwebcando/issues/24)

**Comment by miguelao** [comment]
Ping. Is there anything missing, can I help in any way...?

**Comment by Fyrd** [comment]
Sorry, am on vacation. Looks good though, thanks!

**Pull request merged**